core-war/src/main/java/org/silverpeas/web/jobdomain/servlets/JobDomainPeasRequestRouter.java

```diff
@@ -79,6 +81,12 @@
     ComponentRequestRouter<JobDomainPeasSessionController> {
 
   private static final long serialVersionUID = 1L;
+  private static final String DOMAIN_CREATE_FUNCTION = "domainCreate";
+  private static final String DOMAIN_SCIM_CREATE_FUNCTION = "domainSCIMCreate";
+  private static final String DOMAIN_SQL_CREATE_FUNCTION = "domainSQLCreate";
+  private static final String DOMAIN_DELETE_FUNCTION = "domainDelete";
+  private static final String DOMAIN_SCIM_DELETE_FUNCTION = "domainSCIMDelete";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "DOMAIN_SCIM_DELETE_FUNCTION" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)